### DISCUSSION TITLE
EVA dataworth question

### DISCUSSION THREAD

**Discussion by laat0003** [gmdsi/GMDSI_notebooks#152] [Q&A] [answered]
Hi,

If I am going to evaluate the worth of new data that is ‘as of yet uncollected’ using EVA, then do the actual values that I use, in this case a hypothetical timeseries, influence the uncertainty reduction? For example, if I use a static signal vs one with a stress response included.

Cheers,

**Comment by rhugman** [comment]
Hey @laat0003,

What do you mean by "values that I use"? The idea is that the observation values used to calculate the cross-covariance are obtained from an ensemble of model runs that simulate your forecast period. These outputs/observations will be influenced by the model's behavior in the forecast period. In that regard, the data worth implicitly assumes that the simulated forecast period is representative (is that what you mean?).

So, for example, if you simulate a forecast scenario where all forcings are constat, versus a scenario where forcings vary over time, you may (probably will) get a different estimate of dataworth for each case. This is because there may be different information content generated from fluctuating signals for example.

**Reply by laat0003** [reply]
Thanks @rhugman and @mnfienen for your replies,

This is along the lines of what I was thinking. Apologies if the question seems lacking in sense, and knowing me, that's very likely. I've been thinking about accounting for conceptual model uncertainty as much as possible and how best to go about doing this. More specifically, demonstrating that the model I am using and its prior adequately capture alternative conceptual models of the system. My question was heading toward figuring out if you could use an EVA data worth analysis to demonstrate this and/or as a way to fine tune your prior to ensure that you adequately account for an alternative hypothesis of the system you are simulating.

**Reply by jtwhite79** [reply]
Can you give us an example of the conceptual uncertainty you are grappling with?  Is it geologic structure?  Or processes?  Or?

**Reply by laat0003** [reply]
Thanks @jtwhite79,

More often than not the alternative conceptualizations relate to questions about boundary conditions (source or no-source) and geologic structure.  So for one example, if a boundary can be source then we should see A happen but if it isn't then we should see B happen BUT we may also see B if unit X is discontinuous. Another example a recent project, that you already gave some of your time to, was a fractured rock system where one conceptual model had structural features as competent barriers (known to exist and mapped but unknown hydraulic behavior) exerting significant control on hydraulic heads allowing the bulk properties to be more transmissive but it was also conceivable that the system bulk properties controlled the heads and the features functioned as preferential flow paths. Very different outcomes given the question being asked. I guess this is me lacking intuition about when I should use separate models and analyses for each conceptualization vs. when I can account for alternative conceptual models through creative use of parameterisation and  an adequate prior.

**Reply by jtwhite79** [reply]
Got it.  Was just curious what kind of conceptual uncertainty you are dealing with...

**Comment by mnfienen** [comment, marked as answer]
Hi @laat0003  and @rhugman 

It's my understanding that the covariance is relating model outputs in the potential history matrching period to model outputs in the forecast time. As a result, it's only the model output values that matter - not actual observation values. The model outputs should be collocated in time/space with existing or potential observation values, but the values of those observations are not used. @rhugman  can you confirm that?
